gitbook-docs/security.md

```diff
@@ -112,6 +127,93 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
```

Comment:
How does a client now how often it should validate the token?

Reply:
@tkurki I was wondering that whilst implementing security in a client. Does Node server currently send an expiration, or do they simply not expire until server is restarted?

Reply:
In my case the server handles this automatically for http/ws, so the client doesnt need to do anything. For other transports the token is validated on every incoming message, but that wont cover a long  running subscription.
The expiry is in the token, but I had brain fade, the client cant read that. 
Need to send 'expiry' in the login reply.

Reply:
If we define a standard way to validate we should define a standard way to inform the client about the need.

A client may not have a real time clock, so it should be time to live, not a timestamp?

Reply:
Added `expiry`, in seconds